### PR TITLE
Add queue list styles and functionality

### PR DESCRIPTION
**Pull request by keefyhub**
- Add general styles for queue list
- Add conditional loading of vote/delete buttons based on user
- Add test user to index.js
- Change Search Results titles to be uppercase
- Remove justify content from `.user` so I could reuse it in the queue list (does not seem to have affected the header)
- Hide Up, Down and Delete text, can be removed if needed

Will need tp add a few more changes, including:
- fixed hieght for the queue list with a scroll
- text overflow on track titles
- add class for vote buttons once they have been clicked

screen shot 2016-05-03 at 10 44 58

### CODE REVIEW

File: app/javascripts/components/queue_item.jsx [hunk 1]
```diff
@@ -2,34 +2,71 @@ var QueueItem = React.createClass({
   voteUp: function() {
     socket.emit('vote_up', this.props.item.id)
   },
+  voteUpButton: function() {
+    if(this.props.item.user.user_id != USER_ID) {
```
Comment (harryfinn) [inline app/javascripts/components/queue_item.jsx]: are `user_id` & `USER_ID` both the same type, assume they should be? perhaps worth checking type as well with `!==`

File: app/javascripts/components/queue_item.jsx [hunk 2]
```diff
@@ -2,34 +2,71 @@ var QueueItem = React.createClass({
...
+          <span>Up</span>
+        </div>
+      )
+    }
+  },
   voteDown: function() {
     socket.emit('vote_down', this.props.item.id)
   },
+  voteDownButton: function() {
+    if(this.props.item.user.user_id != USER_ID) {
```
Comment (harryfinn) [inline app/javascripts/components/queue_item.jsx]: same as above

File: app/javascripts/components/queue_item.jsx [hunk 3]
```diff
@@ -2,34 +2,71 @@ var QueueItem = React.createClass({
...
+          <span>Down</span>
+        </div>
+      )
+    }
+  },
   getDuration: function() {
     return this.props.item.duration
   },
   deleteButton: function() {
     if(this.props.item.user.user_id == USER_ID) {
```
Comment (harryfinn) [inline app/javascripts/components/queue_item.jsx]: same as above `===`?

File: app/stylesheets/components/_queue-item.scss [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+    &::before {
+      @include vertical-center(absolute);
+      content: '\f017';
+      font-family: FontAwesome;
+      left: 10px;
+    }
+  }
+
+  h2 {
```
Comment (harryfinn) [inline app/stylesheets/components/_queue-item.scss]: can we make this a class instead? remember element styles should only exist inside the `elements/` folder

File: app/stylesheets/components/_queue-item.scss [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  &__score {
+    background-color: $color-black;
+    border-radius: 50%;
+    color: $color-grey;
+    font-weight: 900;
+    height: 30px;
+    position: relative;
+    width: 30px;
+
+    span {
```
Comment (harryfinn) [inline app/stylesheets/components/_queue-item.scss]: same here

File: app/stylesheets/components/_queue-item.scss [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  margin: 0 20px;
+  transition: opacity $transition-timing,
+              transform $transition-timing;
+
+  &:hover {
+    opacity: .7;
+    transform: scale(1.2);
+  }
+
+  > span {
```
Comment (harryfinn) [inline app/stylesheets/components/_queue-item.scss]: can we make this a class instead? remember element styles should only exist inside the `elements/` folder

File: index.js [hunk 7]
```diff
@@ -89,6 +89,15 @@ app.get('/logout', function(req, res) {
   res.redirect('/')
 })
 
+app.get('/stranger', function(req, res) {
+  res.render('manager/index', {
+    user: {
+      id: 1,
+      displayName: 'Testing'
```
Comment (george1minus1) [inline index.js]: Should this be committed?

File: app/javascripts/components/queue_item.jsx [hunk 8]
```diff
@@ -2,34 +2,71 @@ var QueueItem = React.createClass({
...
-      <div>
-        <span>{this.props.index}</span>
-        <span>{item.title}</span>
-        <span>{this.getDuration()}</span>
-        <span>{item.user.user_name}</span>
-        <button onClick={this.voteUp}>Up</button>
-        <span>{item.score}</span>
-        <button onClick={this.voteDown}>Down</button>
-        {this.deleteButton()}
+      <div className="queue-item">
```
Comment (harryfinn) [inline app/javascripts/components/queue_item.jsx]: as this is a list of items should this markup perhaps use `li`'s with the parent wrapping using a `ol`?

**Comment by george1minus1** [comment]
This looks to be several commits

File: index.js (hunk 7 above)
Comment (keefyhub) [inline index.js]: @muyjohno did you need this for testing or should I remove it from the commit?

File: app/javascripts/components/queue_item.jsx (hunk 8 above)
Comment (keefyhub) [inline app/javascripts/components/queue_item.jsx]: @harryfinn so change the parent div to an `ol`, then the direct children to `li`, and everything inside remain as `div`, `span` etc?

File: index.js (hunk 7 above)
Comment (muyjohno) [inline index.js]: Remove it I think. You can stash this change if you'd like to use it again for dev, but it shouldn't be committed.

**Comment by muyjohno** [comment]
👍

**Pull request merged**